client/src/components/Table/Table.jsx

```diff
@@ -395,6 +407,8 @@ class Table extends Component {
               <FontAwesomeIcon icon={faGear} />
             </span>
           </td>
+        ) || unconfirmedActions.find(el => el === constants.CONFIG) && (
```

Comment:
```suggestion
        ) || unconfirmedActions.find(el => el === constants.TABLE_CONFIG) && (
```

Should correspond to the constant above